common/app/model/content.scala

```diff
@@ -233,7 +237,52 @@ object Content {
     }
   }
 
-  def make(apiContent: contentapi.Content): Content = {
+  object BodyBlocks {
+    def make(blocks: Option[Blocks]): Seq[BodyBlocks] =
+    blocks.toSeq.flatMap { blocks =>
```

Comment:
I actually do it on purpose a lot, but it's an interesting debate.  I definitely don't want to use the outer one in the inner block so from a technical point of view it's a good idea.  The drawback from a readability point of view is github doesn't understand scala well so it's not clear which one I'm using without looking carefully. But if I think too hard that opens a whole debate about why we store code as text files in the first place... :stuck_out_tongue_closed_eyes: